packages/react/src/themes/teams/components/Menu/menuItemStyles.ts

```diff
@@ -269,8 +280,11 @@ const menuItemStyles: ComponentSlotStylesInput<MenuItemPropsAndState, MenuVariab
       underlined,
       vertical,
       disabled,
+      color,
     } = p
 
+    const colorScheme = v.colorScheme[color ? color : primary ? 'primary' : 'grey']
```

Comment:
this logic keeps repeating: `color || (primary ? 'primary' : COLOR_NAME)` can we extract in function?

Reply:
Yep, pretty ugly, because I am using a prop to decide whether I should pick some color (primary)... Will extract this in a separate function.